### PR TITLE
Add Rust projects

### PR DESCRIPTION
**Pull request by Manishearth**
May also be worth linking to https://www.rustaceans.org/findwork

Is there a way I can mark these as Rust projects so that folks can filter by Rust to find bugs?

### CODE REVIEW

**Comment by djmitche** [comment]
OK, sounds good.  I don't really like the granularity of Firefox either..

**Comment by Manishearth** [comment]
Yeah, but this is a way less granular thing, and like I said these projects are pretty different.

**Comment by fiennyangeln** [comment]
You can add Rust before the project name thus it appears close to each other and then it can be filtered later using the search box.

**Comment by Manishearth** [comment]
Done.

Though, more in general, I was asking for a language-based search like bugsahoy and https://starters.servo.org/ , where the tool understands how different projects tag languages (and understands some projects as "default language X", so you can search what you're familiar with.

As in, not just "I want to work on a project that's a part of Rust" (accomplished by searching for "rust"), but "I want to work on a project that's written in rust" (like servo)

**Comment by fiennyangeln** [comment]
Oh ok understand it! like this is it ? https://www.joshmatthews.net/bugsahoy/?rust=1

**Comment by Manishearth** [comment]
yep!

File: src/data/cargo.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: cargo
```
Comment (fiennyangeln) [inline src/data/cargo.yaml]: can we use pascal case here (first letter capitalized) to make it look similar to other project?

File: src/data/clippy.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+name: clippy
```
Comment (fiennyangeln) [inline src/data/clippy.yaml]: can we use pascal case here (first letter capitalized) to make it look similar to other project?

File: src/data/rustfmt.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+name: rustfmt
```
Comment (fiennyangeln) [inline src/data/rustfmt.yaml]: can we use pascal case here (first letter capitalized) to make it look similar to other project?

**Pull request merged**